test/kixi/unit/schemastore/conformers_test.clj

```diff
@@ -105,6 +105,12 @@
                         (s/conform (conformers/regex? "\\k") "bar"))))
 
 (deftest string-conformer
-  (is (valid? string? (s/conform conformers/string? "hello")))
-  (is (not (valid? string? (s/conform conformers/string? 123))))
-  (is (not (valid? string? (s/conform conformers/string? \g)))))
+  (is (valid? string? (s/conform string? "hello")))
```

Comment:
Hmmm...why have you removed the conformers ns? Surely this will use clojure.core?